src/modules/tasks/global-task-detail/global-task-detail.js

```diff
@@ -1,120 +1,116 @@
-    (function() {
+(function() {
 
-        "use strict";
+    "use strict";
 
-        angular
-            .module("TendrlModule")
-            .component("globalTaskDetail", {
+    angular
+        .module("TendrlModule")
+        .component("globalTaskDetail", {
 
-                restrict: "E",
-                templateUrl: "/modules/tasks/global-task-detail/global-task-detail.html",
-                bindings: {},
-                controller: globalTaskDetailController,
-                controllerAs: "glbTaskDetailCntrl"
-            });
+            restrict: "E",
+            templateUrl: "/modules/tasks/global-task-detail/global-task-detail.html",
+            bindings: {},
+            controller: globalTaskDetailController,
+            controllerAs: "glbTaskDetailCntrl"
+        });
 
-        /*@ngInject*/
-        function globalTaskDetailController($rootScope, $scope, $interval, $state, $stateParams, taskStore, config, utils, clusterStore) {
+    /*@ngInject*/
+    function globalTaskDetailController($rootScope, $scope, $interval, $state, $stateParams, taskStore, config, utils, clusterStore) {
```

Comment:
I don't see the use of "utils" service in this controller, please remove the extra dependencies if there is any.

Reply:
Acknowledge